site/src/pages/TemplatePage/TemplateInsightsPage/TemplateInsightsPage.tsx

```diff
@@ -246,17 +256,26 @@ export const TemplateInsightsPageView: FC<TemplateInsightsPageViewProps> = ({
 							? entitlements?.features.user_limit.limit
 							: undefined
 					}
-					data={templateInsights?.interval_reports}
+					data={templateInsights?.data?.interval_reports}
+					error={templateInsights?.error}
```

Comment:
```suggestion
					data={templateInsights.data?.interval_reports}
					error={templateInsights.error}
```

I think a lot of these question marks are no longer needed given the changes to the types

Reply:
Oh snap missed these, thank you!